indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -418,15 +418,14 @@ def _checkParams(self, params):
         ids = set(request.form.getlist('registration_id'))
         self.registrations = (Registration
                               .find(Registration.id.in_(ids), ~Registration.is_deleted)
-                              .with_parent(self.regform)
-                              .all())
+                              .with_parent(self.regform))
 
 
 class RHRegistrationEmailRegistrantsPreview(RHRegistrationsActionBase):
     """Previews the email that will be sent to registrants"""
 
     def _process(self):
-        registration = self.registrations[0]
+        registration = self.registrations.find_first()
```

Comment:
I don't think that'll work. `find_first` is only defined on the models, not on query objects. `.first()` should work though